protocols/identify/Cargo.toml

```diff
@@ -11,10 +11,10 @@ categories = ["network-programming", "asynchronous"]
 
 [dependencies]
 futures = "0.3.1"
-libp2p-core = { version = "0.26.0", path = "../../core" }
-libp2p-swarm = { version = "0.26.0", path = "../../swarm" }
+libp2p-core = { version = "0.27.0", path = "../../core" }
+libp2p-swarm = { version = "0.27.0", path = "../../swarm" }
 log = "0.4.1"
-prost = "0.6.1"
+prost = "0.7"
```

Comment:
So this supersedes https://github.com/libp2p/rust-libp2p/pull/1911 right?

Reply:
Correct. I had to upgrade `prost` in this pull request as well, as otherwise `prost` and `unsigned-varint` / `asynchronous-codec` would use different versions of `bytes`, which in turn fails to compile.